recipes/libpcap/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conans import AutoToolsBuildEnvironment, tools, ConanFile
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibPcapConan(ConanFile):
+    name = "libpcap"
+    version = "1.9.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/the-tcpdump-group/libpcap"
+    description = "libpcap is an API for capturing network traffic"
+    license = "BSD-3-Clause"
+    topics = ("networking", "pcap", "sniffing", "network-traffic")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "enable_dbus": [True, False],
+        "enable_bluetooth": [True, False],
+        "enable_usb": [True, False],
+        "enable_packet_ring": [True, False],
+        "disable_universal": [True, False]
+    }
+    default_options = {
+        'shared': False,
+        'enable_dbus': False,
+        'enable_bluetooth': False,
+        'enable_usb': False,
+        'enable_packet_ring': False,
+        'disable_universal': False
+    }
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _is_amd64_to_i386(self):
+        return self.settings.arch == "x86" and tools.detected_architecture() == "x86_64"
+
+    def requirements(self):
+        if self.options.enable_usb:
+            self.requires("libusb/1.0.23")
+
+    def build_requirements(self):
+        if self.settings.os == "Linux":
+            if not tools.which("bison"):
+                self.build_requires("bison/3.7.1")
+            if not tools.which("flex"):
+                self.build_requires("flex/2.6.4")
+
+    def system_requirements(self):
+        if self.settings.os == "Linux":
+            arch = ":i386" if self._is_amd64_to_i386() else ""
+            package_list = []
+            if self.options.enable_dbus:
+                package_list.extend(["libdbus-glib-1-dev%s" % arch, "libdbus-1-dev"])
+            if self.options.enable_bluetooth:
+                package_list.append("libbluetooth-dev%s" % arch)
+            if self.options.enable_packet_ring:
+                package_list.append("libnl-genl-3-dev%s" % arch)
+            if package_list:
+                package_tool = tools.SystemPackageTool(conanfile=self)
+                package_tool.install(packages=" ".join(package_list), update=True)
```

Comment:
I get an error:
[HOOK - conan-center.py] pre_export(): ERROR: [SYSTEM REQUIREMENTS (KB-H032)] The method 'SystemPackageTool.install' is not allowed in the recipe. (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H032)

How should I address it?

Reply:
you should either create a regular conan package for these requirements, or create a system package for each of these requirements (in the same vein as https://github.com/conan-io/conan-center-index/blob/master/recipes/opengl/all/conanfile.py), or remove the corresponding option

Reply:
What would you recommend to do?
Removing the options seems to be quite easy, but I'm not sure how useful they are.

Reply:
If the pcap is usable without these options, I guess you can just remove them, and wait until someone raises an issue, or makes a pull request.
DISCLAIMER: I don't know this library at all

Reply:
I made the original version for Bincrafters. pcap is used to analyze data packages, (e.g. networking). My Recommendation is removing those options and add a TODO note in this recipe:

# TODO: Add dbus-glib when available
# TODO: Add libbluetooth when available
# TODO: Add libnl-genl when available